Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -2506,12 +2506,14 @@ class PropertyWrapperFontSizeAdjust final : public PropertyWrapperGetter<std::op
 
     void blend(RenderStyle& destination, const RenderStyle& from, const RenderStyle& to, const CSSPropertyBlendingContext& context) const final
     {
-        auto blendedFontSizeAdjust = [&]() -> std::optional<float> {
+        auto blendedFontSizeAdjust = [&]() -> std::optional<FontSizeAdjust> {
             if (context.isDiscrete)
                 return (!context.progress ? from : to).fontSizeAdjust();
             ASSERT(from.fontSizeAdjust().has_value() && to.fontSizeAdjust().has_value());
-            auto blendedAdjust = blendFunc(from.fontSizeAdjust().value(), to.fontSizeAdjust().value(), context);
-            return std::max(blendedAdjust, 0.0f);
+            auto blendedAdjust = blendFunc(*(*from.fontSizeAdjust()).value, *(*to.fontSizeAdjust()).value, context);
+            // FIXME: Handle interpolation between metrics.
```

Comment:
```suggestion
            // FIXME: Handle interpolation between metrics (rdar://problem/107049799).
```

It is linked to https://bugs.webkit.org/show_bug.cgi?id=254266

Reply:
Thanks.